.github/workflows/run_test_from_pr.yml

```diff
@@ -0,0 +1,55 @@
+name: Run (SLOW) desired tests on our runner from a PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number: 
+        description: 'PR number'
+        required: true
+      docker_image:
+        default: 'diffusers/diffusers-pytorch-cuda'
+        description: 'Name of the Docker image'
+        required: true
+      test_command:
+        description: 'Test command to run (e.g.: `pytest tests/pipelines/dit/`). Any valid pytest command can be provided.'
+        required: true
```

Comment:
Would it make sense to have a default here?

Reply:
I think not because having a default here wouldn't really help the case IMO.